### PR TITLE
Hit API

### PR DESCRIPTION
**Pull request by SFarida**
## This PR has the following updates
- creates a game with a post request to `/games/` endpoint
- stores game id in local storage
- add scores to the game created using the game id stored via a post request to `/games/${gameId}/scores/` endpoint.
- get/refresh scores to the game created using the game id stored via a get request to `/games/${gameId}/scores/` endpoint.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+/* eslint-disable import/extensions */
```
Comment (elisha2kyakpo1) [inline src/index.js]: - [ ]  Linters are used to analyze the static source code of your software project to look for problems and flag them. This is the reason it is not good to disable linters. Kindly enable the linter and fix the problem.

File: src/modules/service.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
+/* eslint-disable import/extensions */
+/* eslint-disable no-unused-vars */
```
Comment (elisha2kyakpo1) [inline src/modules/service.js]: - [ ] Let's do the same here. Kindly enable linters to help us have a product that is error-free.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @SFarida, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights

✔️ The PR title and description look good and professional.
✔️ No linter errors.
✔️ Descriptive readme file.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (SFarida) [reply src/index.js]: Hello @elisha2kyakpo1 thanks for the review I correct the .eslintrc.json file as I checked and realised it was not up to date with the one here https://github.com/microverseinc/linters-config/blob/master/html-css-js/.eslintrc.json

**Review by binhussen — APPROVED** [review]
Hi @SFarida , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**